views/check-your-answers.html

```diff
@@ -51,7 +51,7 @@ <h1 class="govuk-heading-xl">Check your answers before sending your application<
         {{ govukButton({
           text: "Continue to payment",
           attributes: {
-            id: submit
+            "id": "submit"
```

Comment:
Change looks fine but just wondering if this addresses this JIRA ticket - https://companieshouse.atlassian.net/browse/ROE-780? If yes, suggest you grab that ticket and also add ref number to this PR. If not, all good. Will approve and let you decide.